### DISCUSSION TITLE
Problem adding system items and clothing after LL update of region servers about two weeks ago

### DISCUSSION THREAD

**Discussion by stonejohnson** [cinderblocks/radegast#40] [General]
It seems that LL caused a problem two weeks ago when they updated their simulator software. Now inventory of system items (e.g., shape, skin, hair, eyes) appears inconsistent (Current Outfit folder is empty no matter what) and Rad seems unable to load system items which is clobbering avatars (turning them into orange clouds).  I guess it is somehow an easy fix because Corrade was affected in the same way but updated their codes quickly (but the fix is beyond me!).  The workaround appears to be to never change system items on an avatar in Radegast.  Attachments seem to be added/removed/inventoried ok.  Thanks!

**Comment by cinderblocks** [comment]
Open this under https://github.com/cinderblocks/radegast/issues with any details you have about what changed and the fix you mentioned.
